osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayerMatchSubScreen.cs

```diff
@@ -219,8 +218,8 @@ public void TestModSelectKeyWithNoAllowedMods()
 
             AddStep("press toggle mod select key", () => InputManager.Key(Key.F1));
 
-            AddWaitStep("wait some", 3);
-            AddAssert("mod select not shown", () => this.ChildrenOfType<ModSelectOverlay>().Single().State.Value == Visibility.Hidden);
+            // AddWaitStep("wait some", 3);
```

Comment:
Probably just remove this completely?

Reply:
whoops, I didn't mean to commit this change